src/runtime_src/core/pcie/driver/linux/xocl/xocl_xclbin.c

```diff
@@ -123,6 +123,26 @@ static int mpsoc_xclbin_post_download(xdev_handle_t xdev, void *args)
 	return 0;
 }
 
+static int xgq_xclbin_pre_download(xdev_handle_t xdev, void *args)
+{
+	return 0;
+}
+
+static int xgq_xclbin_download(xdev_handle_t xdev, void *args)
+{
+	struct xclbin_arg *arg = (struct xclbin_arg *)args;
+	int ret;
```

Comment:
lines 134 and 135 can be combined together or simply replace ret with return.

Reply:
fixed